kotkt.nl/content/ocw/_index.nl.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Open Cultuur Week"
+---
+
+{{< articleimage
+    url="/images/ocw2020.png" 
+>}}
+
+Van 9 september tot en met 11 september organiseren we een paar activiteiten weer alle studenten welkom zijn om mee te doen.
+Deze activiteiten zijn een deel van de [Open Cultuur Week](https://studentencultuur.nl/OCW/) waar alle Studenten Cultuurverenigingen van Eindhoven de deuren open zetten voor nieuwe leden.
+
+{{< aside >}}
+  {{< banner rotate="-5" padding="2px">}}
+      Vanwege COVID-19 is het verplicht om je aan te melden om mee te doen met een van de activiteiten.
+      <a href="https://forms.gle/f1kBn4c8SWdf1GBQ9">Meld je hier aan</a>
+   {{< /banner >}}
+{{</ aside >}}
+
+Zoals iedereen moeten we rekening houden met COVID-19 en hebben we regels opgesteld om ervoor te zorgen dat onze activiteiten veilig plaats kunnen vinden.
+Je kan [hier](https://beta.kotkt.nl/nextcloud/s/x5p6TNtqqzLHiqp/download) onze regels vinden, we raden aan ze door te lezen voordat je langskomt.
+Onderdeel van deze regels is dat **je verplicht bent om je aan te melden** voor onze activiteiten. Niet-leden kunnen zich [hier](https://forms.gle/f1kBn4c8SWdf1GBQ9) aanmelden.
+
+Dinsdag 9 September 19:30: Bordspelavond
+----
+
+Iedere week organiseren de Knights een bordspelavond waar we samen bordspellen spelen.
+
+Zie de [hoofdwebsite](/#boardgames) voor meer informatie over bordspellen.
+
+Woensdag 10 September 19:30: Tabletop Roleplay avond
+----
+
+Eens per maand organiseren de Knights een Tabletop Roleplay avond waar onze storytellers een one-shot sessie runnen.
+Zij regelen een roleplay systeem (denk aan Dungeons & Dragons, Fate of iets anders), karakters en samen spelen jullie door het verhaal.
+Deze sessies duren een avond en aan het eind van de avond eindigt het verhaal, dus je zit niet vast aan een hele campagne als je meedoet met deze activiteit.
+
+Er worden wel roleplay-campaigns bij de Knights gespeeld overigens, als je daarin geinteresseerd bent: vraag het aan een bestuurslid of stuur een mail naar info@kotkt.nl.
+
+Donderdag 11 September 19:30: LARP zwaardvecht training
+----
+
+Iedere twee weken organiseren de Knights een LARP zwaardvechttraining.
+Tijdens de training oefen je met LARP-safe zwaarden.
+Omdat dit de eerste training van het jaar is, gaan we focussen op beginners, dus als je nog niet weet hoe je een zwaard moet vasthouden: dit is je kans!
+
+Naast de zwaardvechttraining organiseren we ook een paar LARPS, voor meer informatie daarover [zie de hoofdsite](/#larp)
+
+Niet in staat langs te komen?
+---
```

Comment:
this should be 4 dashes if you want to keep the dashes

Reply:
Yeah, I should probably switch to `##` for consistency with the rest of the site